src/Classes/ImportTab.lua

```diff
@@ -100,6 +100,9 @@ You can get this from your web browser's cookies while logged into the Path of E
 		self.charImportMode = "GETACCOUNTNAME"
 		self.charImportStatus = "Idle"
 	end)
+	self.controls.sessionPrivacySettings = new("ButtonControl", {"LEFT",self.controls.sessionCancel,"RIGHT"}, 8, 0, 120, 20, "Privacy Settings", function()
+		os.execute('start https://www.pathofexile.com/my-account/privacy')
```

Comment:
Technically this URL is only correct for non-Garena, non-Tencent accounts.  You should also use the `OpenURL` API function instead.

Reply:
ah, I was looking for some function like that but couldn't find one

I didn't do anything about garena/tencent accounts. LMK if there's something you had in mind